engine/src/cmd/csv.cpp

```diff
@@ -291,9 +291,9 @@ CSVRow::CSVRow(CSVTable *parent, const string &key) {
     iter = parent->rows[key] * parent->key.size();
 }
 
-CSVRow::CSVRow(CSVTable *parent, unsigned int i) {
+CSVRow::CSVRow(CSVTable *parent, unsigned int which) {
```

Comment:
I recently disabled CSV support in the game. Not sure which PR.
Tracing CSVRow, it's not actually used by anything.

Reply:
OK, I can take it out.

Reply:
Deleted.